src/PasswordInput/Examples.md

```diff
@@ -0,0 +1,15 @@
+### Examples
+
+```js
+initialState = { value: '' };
+
+const onChange = value => setState({ value });
+
+<form className='ui form'>
```

Comment:
this is a left-over from when we used semantic's css, you can remove the `className` and, if you want, the `form` as well